spotipy/client.py

```diff
@@ -34,6 +34,67 @@ class Spotify(object):
     """
     max_retries = 3
     default_retry_codes = (429, 500, 502, 503, 504)
+    country_codes = [
```

Comment:
I searched for awhile for a comprehensive list of available markets supported by Spotify - I was able to find [this](https://github.com/spotify/web-api/issues/442#issuecomment-315400463) issue over on the spotify/web-api repo. It's a bummer Spotify wont provide an endpoint or at least a list of markets that can be searched, so I figured this would be next best - and it could be updated as needed.